core/src/main/java/com/graphhopper/util/Instruction.java

```diff
@@ -141,6 +141,10 @@ public PointList getPoints() {
         return points;
     }
 
+    public void setPoints(PointList points) {
+        this.points = points;
```

Comment:
Not easily. The issue is that we have to change the points, once the instruction is created. Changing this might be complicated, but not necessarily impossible.

Once the instructions and PathDetails are calculated, we do the simplification. When simplifying, we need to change the points. I think changing this would require quite a lot overhead.